src/main/kotlin/org/rust/lang/core/resolve2/FacadeResolve.kt

```diff
@@ -336,17 +336,18 @@ private fun VisItem.toPsi(defMap: CrateDefMap, project: Project, ns: Namespace):
     return when (containingModOrEnum) {
         is RsMod -> {
             if (ns == Namespace.Macros) {
-                val macros = containingModOrEnum.expandedItemsCached.macros
+                val items = containingModOrEnum.expandedItemsCached
+                val macros = items.macros
                     .filter { it.name == name && matchesIsEnabledByCfg(it, this) }
                 // TODO: Multiresolve for macro 2.0
                 if (macros.isNotEmpty()) return listOf(macros.last())
 
-                containingModOrEnum
+                items.rest
                     // Note: name of bang and attribute proc macros is same as name of item
```

Comment:
Looks like this comment is also not needed now (because we properly use `procMacroName`)